testing/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+cmake_minimum_required(VERSION 3.15)
+
+file(GLOB SOURCE_FILES "*.cc")
```

Comment:
please don't use glob, this is known as buggy. prefer list files

Reply:
I copied it from game folder. Should we change it there too?

Reply:
well actually, I would agree with it, but as there are quite a lot of file in game/*c, I'm not sure everybody will agree... That said, in your case, you should have quite a few files, so should be ok.

Reply:
Done